resources/dashboards/Traffic-Profile-Dashboard.json

```diff
@@ -0,0 +1,11 @@
+{
+   "hits": 0, 
+   "timeRestore": false, 
+   "description": "", 
+   "title": "Traffic Profile Dashboard", 
```

Comment:
I think the names of these two dashboards are switched...

This dashboard displays a table with Src & Dest IPs, as well as visualizations about the 'top talkers' - the Src and Dest IPs that account for the most sessions and highest metadata impact.

The other dashboard shows stats about the volume of traffic, the size of the average JSON insertion, etc.

The first description sounds like a dashboard centered around Endpoints, and the second description sounds like a dashboard centered around your traffic profile.

Reply:
Agreed @DylanSchneider